src/elements/cartoon/cartoonDataTransformer.ts

```diff
@@ -1,40 +1,72 @@
-import type { Breakpoint } from "@guardian/src-foundations";
 import { undefinedDropdownValue } from "../../plugin/helpers/constants";
 import type { FieldNameToValueMap } from "../../plugin/helpers/fieldView";
 import type { Asset } from "../helpers/defaultTransform";
 import type { MainImageData } from "../helpers/types/Media";
 import type { TransformIn, TransformOut } from "../helpers/types/Transform";
 import type { cartoonFields } from "./CartoonSpec";
```

Comment:
The CartoonSpec doesn't _need_ updating as the `transformElementIn` function returns typeof cartoonFields. That said, it might be nice to rename the desktop and mobile image arrays to be consistent with our new viewport sizes.

Reply:
Good shout @rebecca-thompson, I've renamed them `largeImages` and `smallImages`